setup.py

```diff
@@ -24,13 +30,13 @@ def validateReST(text):
 else:
     import docutils.utils
     import docutils.parsers.rst
-    import StringIO
+    from io import StringIO
     def validateReST(text):
         doc = docutils.utils.new_document('validator')
         # our desired settings
         doc.reporter.halt_level = 5
         doc.reporter.report_level = 1
-        stream = doc.reporter.stream = StringIO.StringIO()
+        stream = doc.reporter.stream = StringIO()
```

Comment:
`io.StringIO()` on Python 2 rejects native strings and insists on Unicode.  Is that what docutils.reporter prints on Python 2?

Reply:
Will check it and maybe provide a test

Reply:
You can always add a conditional that would use `StringIO()` on Python 3 and `BytesIO()` on Python 2 -- the default settings for `new_document` and its `reporter` will be `ascii` in docutils, I'm afraid.